### PR TITLE
[FIX] stock_barcodes: Do not distribute lots in inventory if a lot is selected in wizard

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

ping @carlosdauden @chienandalu

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Red CI

File: stock_barcodes/wizard/stock_barcodes_read.py [hunk 1]
```diff
@@ -91,7 +91,6 @@ def _compute_is_manual_qty(self):
         for rec in self:
             rec.is_manual_qty = rec.option_group_id.is_manual_qty
             rec.is_manual_confirm = rec.option_group_id.is_manual_confirm
-            rec.auto_lot = rec.option_group_id.auto_lot
```
Comment (chienandalu) [inline stock_barcodes/wizard/stock_barcodes_read.py:94]: I'd say test fail because of this change

**Comment by sergio-teruel** [comment]
@pedrobaeza Changes done!!

**Comment by pedrobaeza** [comment]
OK, thanks. @chienandalu must say if OK.

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
CI is again red

**Comment by sergio-teruel** [comment]
@pedrobaeza Changes done!!

**Review by chienandalu — APPROVED** [review]

**Pull request merged**